foodsaving/utils/tests/test_email_utils.py

```diff
@@ -1,14 +1,18 @@
-from rest_framework.test import APITestCase
+from unittest import TestCase
+
+import pytz
+from django.utils import timezone
 
 from config import settings
 from foodsaving.groups.factories import GroupFactory
 from foodsaving.invitations.models import Invitation
 from foodsaving.userauth.models import VerificationCode
 from foodsaving.users.factories import UserFactory
 from foodsaving.utils import email_utils
+from foodsaving.utils.email_utils import time_filter, date_filter
 
 
-class TestEmailUtils(APITestCase):
+class TestEmailUtils(TestCase):
```

Comment:
It seems the tests fail because `pytest` doesn't allow a `unittest.TestCase` to access the database, but a `django.test.TestCase`: https://pytest-django.readthedocs.io/en/latest/helpers.html#pytest-mark-django-db-transaction-false-request-database-access

Reply:
Ah, makes sense!